hack/tas-kind-cluster.yaml

```diff
@@ -0,0 +1,107 @@
+---
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+  - role: control-plane
+    kubeadmConfigPatches:
+      - |
+        kind: ClusterConfiguration
+        apiVersion: kubeadm.k8s.io/v1beta3
+        scheduler:
+          extraArgs:
+            v: "2"
+        controllerManager:
+          extraArgs:
+            v: "2"
+        apiServer:
+          extraArgs:
+            enable-aggregator-routing: "true"
+            v: "2"
+  - role: worker
+    labels:
+      cloud.provider.com/node-group: tas-group
+      cloud.provider.com/topology-block: b1
+      cloud.provider.com/topology-rack: r1
+    kubeadmConfigPatches:
+      - |
+        kind: InitConfiguration
+        nodeRegistration:
+          kubeletExtraArgs:
+            v: "2"
```

Comment:
isn't v2 the default one? In any case it seems not really needed. Let's add if we really hit some issues with kubelet